examples/specs/layer_bar_waterfall_chart.vl.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "data": {
+    "values": [
+      {"label": "Start", "amount": 4000, "order": 1},
+      {"label": "Jan", "amount": 1707, "order": 2},
+      {"label": "Feb", "amount": -1425, "order": 3},
+      {"label": "Mar", "amount": -1030, "order": 4},
+      {"label": "Apr", "amount": 1812, "order": 5},
+      {"label": "May", "amount": -1067, "order": 6},
+      {"label": "Jun", "amount": -1481, "order": 7},
+      {"label": "Jul", "amount": 1228, "order": 8},
+      {"label": "Aug", "amount": 1176, "order": 9},
+      {"label": "Sep", "amount": 1146, "order": 10},
+      {"label": "Oct", "amount": 1205, "order": 11},
+      {"label": "Nov", "amount": -1388, "order": 12},
+      {"label": "Dec", "amount": 1492, "order": 13},
+      {"label": "End", "amount": 0, "order": 14}
+    ]
+  },
+  "transform": [
+    {
+      "window": [{"op": "sum", "field": "amount", "as": "sum"}],
+      "sort": [{"field": "order", "order": "ascending"}]
+    },
+    {
+      "calculate": "datum.label == 'End' ? 0 : datum.sum - datum.amount",
+      "as": "last"
+    },
+    {
+      "calculate": "datum.label == 'End' ? datum.sum : datum.amount",
+      "as": "amount"
+    },
+    {
+      "calculate": "datum.label == 'End' || datum.label == 'Start' ? 'Start/End' : datum.amount >= 0 ? 'increasing' : 'decreasing'",
+      "as": "color"
+    },
+    {"calculate": "datum.sum - 100", "as": "bottomline"}
+  ],
+  "layer": [
+    {
+      "mark": "bar",
```

Comment:
Why is there only one layer? Add labels like the community example in the gallery?